filecoin-proofs/tests/paramfetch/prompts_to_fetch.rs

```diff
@@ -146,3 +155,41 @@ fn fetches_vk_even_if_sector_size_does_not_match() -> Result<(), FailureError> {
 
     Ok(())
 }
+
+#[test]
+fn invalid_json_path_produces_error() -> Result<(), FailureError> {
+    let mut session = ParamFetchSessionBuilder::new(Some(PathBuf::from("/invalid/path")))
+        .with_session_timeout_ms(1000)
+        .build();
+
+    session.exp_string("fatal error: json file '/invalid/path' does not exist")?;
+
+    Ok(())
+}
+
+#[test]
+fn invalid_json_produces_error() -> Result<(), FailureError> {
+    let manifest_pbuf = tmp_manifest(None)?;
+
+    let mut file = File::create(&manifest_pbuf)?;
+    file.write_all(b"invalid json")?;
+
+    let mut session = ParamFetchSessionBuilder::new(Some(manifest_pbuf))
+        .with_session_timeout_ms(1000)
+        .build();
+
+    session.exp_string("fatal error: expected value at line 1 column 1")?;
```

Comment:
I would like to see a string like: `fatal error: file with path '/invalid/path' did not contain valid manifest JSON`